package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
 	"name": "magicmirror",
-	"version": "2.30.0",
+	"version": "2.31.0-develop",
 	"lockfileVersion": 3,
 	"requires": true,
 	"packages": {
 		"": {
 			"name": "magicmirror",
-			"version": "2.30.0-develop",
+			"version": "2.31.0-develop",
```

Comment:
@rejas seems on master is still "2.30.0-develop" instead of "2.30.0", maybe a problem when releasing the npm package?

Reply:
i missed that on the change

Reply:
Haven't done the npm release yet, but this shouldn't be a problem.